### DISCUSSION TITLE
how to install a project locally?

### DISCUSSION THREAD

**Discussion by kiberlain** [retypeapp/retype#14] [Q&A] [answered]
I can not deploy to vercel hosting

**Comment by geoffreymcgill** [comment]
Hi @kiberlain, 

Can you provide some additional details on your local Retype setup and what might be required for Vercel? Any details on what exactly is not working would be helpful.

I don't have experience with Vercel, but when Retype builds your project, everything that is required to host your project on a web server is sent to the [`output`](https://retype.com/configuration/project/#output) folder.

The built website is a static html website and no specific server-side technology is required, other than the ability to host static files.

If you setup the [Retype GitHub Action](https://retype.com/guides/github_actions/) to build your project on each new push, the default `output` location of the GitHub Action is the `retype` branch. Maybe Vercel could be pointed to that `retype` branch to wait for changes?

Hope this helps.

**Comment by fabriciomurta** [comment, marked as answer]
In case you are serving the files elsewhere not the website root, you need to [set retype.json's `base` config](https://retype.com/configuration/project/#base).

So if the local server path is something like `wwwroot/docs`, so public address would be http://myhost.com/docs), the retype.json file would need to have `"base": "docs"`  to honor that path.

In other words, it should point to your documentation's root relative to the website's.

**Comment by fabriciomurta** [comment]
Hey, we've moved your issue into a thread, so don't mind the "issue closed" message; we just converted it into a discussion.
